dockerfile

```diff
@@ -0,0 +1,26 @@
+# pull official base image
+FROM node:14.17-alpine AS builder
+
+# set working directory
+WORKDIR /parity-bridges-ui
+
+RUN apk update
+RUN apk add git
+
+# add `/parity-bridges-ui/node_modules/.bin` to $PATH
+ENV PATH /parity-bridges-ui/node_modules/.bin:$PATH
+
+# install app dependencies
+COPY . .
+RUN yarn && yarn build
```

Comment:
To speed up build times it's common to split depenedencies installation from actual build process, so that dependencies can be cached.
```
COPY package.json .
COPY yarn.lock .
RUN yarn
COPY . .
RUN yarn build
```

This should give much better times when re-building the image and there is no `package.json` changes.